packages/core/test/actor.test.ts

```diff
@@ -1287,7 +1287,7 @@ describe('actors', () => {
             fromPromise(
               () =>
                 new Promise<number>((_, rej) => {
-                  setTimeout(() => rej(errorMessage), 1000);
+                  setTimeout(() => rej(errorMessage), 1);
```

Comment:
just a small speed boost - there was no reason here to wait for the whole second